agent/http.go

```diff
@@ -368,6 +368,9 @@ func (s *HTTPHandlers) wrap(handler endpoint, methods []string) http.HandlerFunc
 				}
 				logURL = strings.Replace(logURL, token, "<hidden>", -1)
 			}
+			httpLogger.Warn("This request used the token query parameter "+
+				"which is deprecated and will be removed in Consul 1.17",
+				"logUrl", logURL)
```

Comment:
@kisunji , @cthain , @roncodingenthusiast : Should we strip the token from the `logUrl`?

Reply:
Does the token secret get stripped from the `logUrl` in the log output? Or does this log message actually include the token secret?

Reply:
Edit: I've been informed that the `logUrl` already obfuscates the token before the log is emitted